### PR TITLE
fix: octoxlabs device function

### PR DESCRIPTION
**Pull request by ogulcanhero**
## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Octox Labs Cyber Security Asset Management platform XSoar Integration update.

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [x] 6.1.0
- [x] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by ogulcanhero** [comment]
I think I filled Contribution Form incorrectly and I reposted form. Is there any chance we can restart pipeline?

**Comment by ShacharKidor** [comment]
> I think I filled Contribution Form incorrectly and I reposted form. Is there any chance we can restart pipeline?

Hi @ogulcanhero, 
We had a problem with the for submission during the weekend - I've just fixed it and fill out your details. 
Thank you.

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Development container for Content. Poetry hash 1.11.0
+// Development container for Content. Poetry hash 022d65df3402ad71ca04cca0b76b759f0874f425330e511e952f4d2ebf9f265f
```
Comment (GuyAfik) [inline .devcontainer/devcontainer.json:1]:
```suggestion
// Development container for Content. Poetry hash 1.11.0
```

**Review by GuyAfik — APPROVED** [review]
@ogulcanhero looks great! Thanks for your contribution

**Comment by ogulcanhero** [comment]
@GuyAfik, Thank you for dedicating your time 😄

**Pull request merged**